packages/components/addon/components/hds/dismiss-button/index.hbs

```diff
@@ -0,0 +1,3 @@
+<button class="hds-dismiss-button" type="button" aria-label="Dismiss" ...attributes>
```

Comment:
[question] Is there a way to allow the aria-label attribute to be internationalized? 
It might also be something that we want to address all at once (as an initiative, for example) across all of our components, though. I guess if we don't do it now, we will definitely need to support it later.

Reply:
Yes, I think it's a good idea to do it across the repo. We captured this in a [Jira issue](https://hashicorp.atlassian.net/browse/HDS-567).